src/data_access/repository.py

```diff
@@ -253,27 +253,72 @@ def swap_positions(self, queue_id, pos_1, pos_2):
         self.cursor.execute("""
             UPDATE queue_members
             SET position = CASE position
-                WHEN :pos_1 THEN -1
-                WHEN :pos_2 THEN -2
+                WHEN :pos_1 THEN -:pos_2
```

Comment:
What's the reason for this change?

Reply:
Both approaches are good, I just would like us to concentrate on more meaningful changes